hdd_tools/DOCS.md

```diff
@@ -18,6 +18,7 @@
 | performance_check | Flag to enable or disable the execution of performance check at startup
 | debug | Flag to enable or disable debugging. Activate this if you want to debug which property from the JSON output of `smartctl` you want to be merged to the sensor.
 | output_file | Log file
+| device_type | Type of block device which `smartctl` will try to use for communication. Default "auto". Check https://www.smartmontools.org/wiki/USB
```

Comment:
I think we need to change the parameter position. We have the "options" for the command up, then performance, debug, output, etc. so better put it for example after `hdd_path`, it seems more natural to me. What do you think?